implicits/src/com/rallyhealth/weepickle/v1/implicits/MacroImplicits.scala

```diff
@@ -543,13 +543,9 @@ private object MacroImplicits {
           .map(i => q"${foundBitSet(i)} != ${if (i < addlBitSets || args.length > 0 && args.length % 64 == 0) -1 else (1L << args.length) - 1}")
           .reduce((a, b) => q"$a || $b")
       }){
-                var i = 0
-                val keys = for{
-                  i <- 0 until ${args.length}
-                  if isMissing(i)
-                } yield i match{
+                val keys = (0 until ${args.length}).withFilter(!isMissing(_)).map{
```

Comment:
```suggestion
                val keys = (0 until ${args.length}).withFilter(isMissing(_)).map{
```

Did `isMissing` got flipped from the original?